dist/21-526EZ-ALLCLAIMS-schema.json

```diff
@@ -1016,6 +1016,9 @@
         }
       }
     },
+    "terminallyIll": {
```

Comment:
The BE is looking for `isTerminallyIll` in the form.

Reply:
sounds good, will update

Reply:
Thanks!